### PR TITLE
Added state checks before action is commited

### PR DESCRIPTION
**Pull request by RishiChillara**

### CODE REVIEW

**Comment by jasondpham** [comment]
Everything looks good to me and is still working. I ran this change locally and noticed an image no longer appeared. I know image uploading is WIP, but is this an intentional change? Also, slight nit - do we need a none status? Is does not seem reachable.

Should be good after addressing those!

File: flutter-app/flutter_drawing_app/starter/lib/drawing_page.dart [hunk 1]
```diff
@@ -7,12 +7,14 @@ import 'package:flutter/material.dart';
...
- enum Status {
-    none,
-    color,
-    linedrawing,
-    upload_image
-  }
+ 
+// Status keeps track of what action the user is trying to execute, default to none on start
+enum Status {
+  none,
```
Comment (jasondpham) [inline flutter-app/flutter_drawing_app/starter/lib/drawing_page.dart:13]: Do we need this?

**Review by jasondpham — COMMENTED** [review]
Left a comment on the PR. LGTM if we can address those

**Comment by sarahukani** [comment]
For now, there being no image at first was an intentional change. Did we want to keep a default image at pop up or tell user to upload an image?

File: flutter-app/flutter_drawing_app/starter/lib/drawing_page.dart (hunk 1 above)
Comment (RishiChillara) [reply flutter-app/flutter_drawing_app/starter/lib/drawing_page.dart:13]: yes we do

**Pull request merged**